orttraining/orttraining/python/training/onnxblock/engine/module.py

```diff
@@ -0,0 +1,90 @@
+from onnxruntime.capi import _pybind_state as C
+from onnxruntime.capi.onnxruntime_pybind11_state import OrtValueVector
+
+
+class Module:
+    """
+    Class for running Training.
+    This class is a wrapper of Module Class.
+    """
+
+    training: bool
+
+    def __init__(self, train_model_uri, ckpt_uri, eval_model_uri=None) -> None:
+        """
+        Initializes Model for Training.
+        __init__ will call an internatl function to create the model.
+        """
+        # TODO : Add support for bytes on train_model_uri and eval_model_uri.
+        self.training = True
+        self._train_model_uri = train_model_uri
+        self._ckpt_uri = ckpt_uri
+        self._eval_model_uri = eval_model_uri
+
+        self._create_training_module()
```

Comment:
Since the paths are not needed after the construction, we can just use them transactionally.
```suggestion
        self._create_training_module(train_model_uri, ckpt_uri, eval_model_uri)
```

Reply:
another approach is to get rid of _create_training_module since it does not really do much